ckanext/privatedatasets/actions.py

```diff
@@ -114,11 +202,11 @@ def package_acquired(context, request_data):
                         context_pkg_update['user'] = user.get('name', '')
 
                         plugins.toolkit.get_action('package_update')(context_pkg_update, dataset)
-                        log.info('Allowed Users added correctly')
+                        log.info('Allowed Users %s correctly') % (message)
                     else:
-                        log.warn('The user %s is already allowed to access the %s dataset' % (user_info['user'], dataset_id))
+                        log.warn('The user %s is already %s from accessing the %s dataset' % (user_info['user'], message, dataset_id))
```

Comment:
"The user aitor is already added from accessing"
"The user aitor is already deleted from accesing" 

???